src/app/public/modules/inline-delete/inline-delete-adapter.service.ts

```diff
@@ -22,9 +22,9 @@ export class SkyInlineDeleteAdapterService {
 
   constructor(
     private coreAdapterService: SkyCoreAdapterService,
-    private rendererFactory: RendererFactory2
+    rendererFactory: RendererFactory2
   ) {
-    this.renderer = this.rendererFactory.createRenderer(undefined, undefined);
+    this.renderer = rendererFactory.createRenderer(undefined, undefined);
```

Comment:
Since the renderer factory is never used outside the constructor, we can reference it as a simple argument (without the access modifier) and minimize the size of the compiled JavaScript bundle.

Reply:
![](https://media.giphy.com/media/3og0IMJcSI8p6hYQXS/giphy.gif)